.changeset/sharp-items-study.md

```diff
@@ -0,0 +1,19 @@
+---
+'@backstage/backend-defaults': minor
```

Comment:
```suggestion
'@backstage/backend-defaults': patch
```
I believe this should be patch unless something is removed in the API. Would be minor if package was 1.x and new features was added.

Reply:
This is patch right? 🤔